### ISSUE TITLE
Datetime for notification

### ISSUE THREAD

**Issue by ChristianKleineidam** [Discordius/Lesswrong2#444]
When looking at the notifications it's often hard to know which one I already read before. It would be great if I would see the datetime of the notifications to inform myself.

**Comment by movationdesign** [comment]
I agree. It's a bit confusing right now.

**Comment by Discordius** [comment]
Yep, seems like a good idea. We already have the data, so doing that should be trivial.
